src/components/DesktopNewsletter/index.jsx

```diff
@@ -0,0 +1,16 @@
+
+
+export default function DesktopNewsletter() {
+    return(
+    <div className="grid h-screen bg-primary place-items-center">
+        <div className="relative h-full w-full items-center justify-center inline-flex">
+            <div className="relative h-4/6 w-1/2 bg-primary-dark rounded-[2.813rem] bottom-10 right-10"/>
+            <div className="absolute h-4/6 w-1/2 pl-7 bg-secondary-light rounded-[2.813rem]"> 
```

Comment:
Instead of doing this can we use h-fit and drop shadow to get the same effect? This way on smaller screens that are still desktops things don't get shoved outside the box, and when the screen is large, the box doesn't have a ton of space around it?
filter: drop-shadow(-90px -60px 4px rgba(0, 0, 0, 0.5));